crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -138,16 +134,25 @@ def bag_item
 
     private
 
+    def repos_dir
```

Comment:
And just using `Pathname` instead of this string makes other parts a lot easier as well.